nms/app/packages/magmalte/app/views/network/FEGServicingAccessGatewayTable.js

```diff
@@ -0,0 +1,135 @@
+
+type ServicingAccessGatewayRowType = {
+  networkId: network_id,
+  networkName: network_name,
+  gatewayId: gateway_id,
+  gatewayName: gateway_name,
+};
+
+/**
+ * Returns a table consisiting of the serviced access gateways alongside
+ * the serviced network in which they are under.
+ */
+export default function ServicingAccessGatewayInfo() {
+  const {match} = useRouter();
+  const enqueueSnackbar = useEnqueueSnackbar();
+  const networkId: string = nullthrows(match.params.networkId);
+  const [servicedAccessGatewaysInfo, setServicedAccessGatewaysInfo] = useState<
+    Array<ServicingAccessGatewayRowType>,
+  >([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const getServicedAccessGatewaysInfo = async (
+      servicedAccessNetworks: Array<feg_lte_network>,
+    ): Promise<Array<ServicingAccessGatewayRowType>> => {
+      const newServicedAccessGatewaysInfo = [];
+      for (const servicedAccessNetwork of servicedAccessNetworks) {
+        const servicedAccessGateways = await FetchGateways({
+          networkId: servicedAccessNetwork.id,
+          undefined,
+          enqueueSnackbar,
+        });
+        //Add the gateways of the serviced network
+        for (const servicedAccessGatewayId in servicedAccessGateways) {
+          const newServicedAccessGatewayInfo: ServicingAccessGatewayRowType = {
+            networkId: servicedAccessNetwork.id,
+            networkName: servicedAccessNetwork.name,
+            gatewayId: servicedAccessGatewayId,
+            gatewayName: servicedAccessGateways[servicedAccessGatewayId]?.name,
+          };
+          newServicedAccessGatewaysInfo.push(newServicedAccessGatewayInfo);
+        }
+      }
+      return newServicedAccessGatewaysInfo;
+    };
+    const fetchServicedAccessGateways = async () => {
+      try {
+        const servicedAccessNetworks = await getServicedAccessNetworks(
+          networkId,
+          enqueueSnackbar,
+        );
+        const newServicedAccessGatewaysInfo = await getServicedAccessGatewaysInfo(
+          servicedAccessNetworks,
+        );
+        setServicedAccessGatewaysInfo(newServicedAccessGatewaysInfo);
+        setIsLoading(false);
+      } catch (e) {
+        enqueueSnackbar?.(
+          'failed fetching servicing access gateway information',
+          {
+            variant: 'error',
+          },
+        );
+      }
+    };
+    fetchServicedAccessGateways();
+  }, [networkId, enqueueSnackbar]);
+  if (isLoading) {
+    return <LoadingFiller />;
+  }
+  return (
+    <div>
+      <ActionTable
+        data={servicedAccessGatewaysInfo}
+        columns={[
+          {title: 'Access Network', field: 'networkName'},
+          {
+            title: 'Access Gateway',
+            field: 'gatewayName',
+            render: currRow => (
+              <Link
+                variant="body2"
+                component="button"
+                onClick={() => {
+                  window.open(
```

Comment:
We have generally used history push pattern instead of window open pattern. For e.g.
https://github.com/magma/magma/blob/d691319dd40e0a7d2822a993f819beca7490e65d/nms/app/packages/magmalte/app/views/equipment/GatewayDetailEnodebs.js#L95

Reply:
Yeah, I tried that but history doesn't work for this case because we are going from one network to another network, the context providers need to run again since they have different data. If we are **moving** from one page to another page **in the same network**, it would work since the already loaded data would be used and don't need to be changed. So, I discussed this with Andrei, last time and he suggested to open a new tab for now because that way we would force to load the data of the new network that we are traveling into.